index.html

```diff
@@ -2,276 +2,313 @@
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- DESC kernels -->
+     <div class="item">
+          <div class="box">
+               <input type="checkbox">
+          </div>
+          <div class="text">
+               Follow the <a href="https://confluence.slac.stanford.edu/x/1_ubDQ" title="link to Confluence">instructions to install DESC jupyter kernels</a>.
+               You will need them to access DC2 data products or to use DESC software during the Collaboration meeting.
+          </div>
+     </div>
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- DC2 -->
+     <div class="item">
+          <div class="box">
+               <input type="checkbox">
+          </div>
+          <div class="text">If you have not used any DC2 data tutorial notebooks before, check out the instructions on this
+               <a href="https://confluence.slac.stanford.edu/x/Xgg4Dg" title="link to Confluence">step-by-step</a> guide. Also, bookmark the
+               <a href="https://confluence.slac.stanford.edu/x/oJgHDg" title="link to Confluence">DC2 Data Product Overview page</a>
+               and the
+               <a href="https://github.com/LSSTDESC/DC2-analysis" title="link to GitHub">DC2-analysis GitHub repo</a> for easy access.
+               Questions? Ask them in the
+               <a href="https://lsstc.slack.com/messages/CBUMARLKC/" title="link to LSSTC Slack">#desc-dc2-users</a> Slack
+               channel.
+          </div>
+     </div>
+
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- Social media -->
+     <h2>Social media</h2>
+     <!-- -->
+     <div class="item">
+          <div class="box">
+               <input type="checkbox">
+          </div>
+          <div class="text">If you use social media professionally, please use the
+               <a href="https://twitter.com/hashtag/lsstdesc" title="link to Twitter">#lsstdesc</a>
+               hashtag when reporting highlights from the meeting! Remember the Code of Conduct mentioned above!
+          </div>
+     </div>
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <div class="footer">
+          <p>This site uses cookies to keep track your checklist progress. The data are stored in your browser locally only.</p>
+          <p>If you have questions or suggestions about this page, please
+               <a href="https://github.com/LSSTDESC/pre-meeting-checklist/issues" title="link to GitHub">open an issue</a> or contact
+               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:agkim@lbl.gov?Subject=DESC%20Meeting%20Checklist" target="_top">Alex Kim</a>.</p>
```

Comment:
Change the contacts here?

Reply:
you and alex are the ones with admin privileges to the repo so i'm not sure if it makes sense to change the contact info for "questions or suggestions about **this** page".